### ISSUE TITLE
[BUG] _Training Fails

### ISSUE THREAD

**Issue by swome** [deepmodeling/deepmd-kit#679]
**Summary**

DeepMD-Kit commit number cb2c118, may possibly fail to decrease training error. After tens of thousands steps of training the RMSE of energy remains 1E+02 orders of magnitude. This happens occasionally.

tail lcurve.out
 172000      1.57e+03      3.71e+02      1.21e-01      2.55e+00    1.1e-04
 173000      6.42e+03      3.54e+02      1.64e+01      1.66e+03    1.1e-04
 174000      1.59e+03      3.76e+02      4.61e-03      2.08e+00    1.1e-04
 175000      1.74e+03      3.76e+02      2.57e-01      2.06e+00    1.1e-04
 176000      2.36e+03      3.76e+02      3.05e-01      1.66e+00    1.0e-04
 177000      1.75e+03      3.76e+02      1.85e-01      1.94e+00    1.0e-04
 178000      1.75e+03      3.76e+02      2.06e-01      1.97e+00    1.0e-04
 179000      1.59e+03      3.75e+02      1.21e-02      1.79e+00    1.0e-04
 180000      1.75e+03      3.76e+02      1.62e-01      1.67e+00    9.9e-05
 181000      2.37e+03      3.76e+02      1.79e-02      2.12e+00    9.9e-05

Not happening on older  version b30c09e.

**Deepmd-kit version, installation way, input file, running commands, error log, etc.**
<version>: cb2c118

<input.json>
{
    "_comment": "that's all",
    "model": {
        "type_map": [
            "Li"
        ],
        "descriptor": {
            "type": "hybrid",
            "list": [
                {
                    "type": "se_a",
                    "sel": [
                        700
                    ],
                    "rcut_smth": 0.0,
                    "rcut": 9.0,
                    "neuron": [
                        20,
                        40,
                        80
                    ],
                    "resnet_dt": false,
                    "axis_neuron": 16,
                    "_comment": " that's all"
                },
                {
                    "type": "se_at",
                    "sel": [
                        48
                    ],
                    "rcut_smth": 0.0,
                    "rcut": 3.0,
                    "neuron": [
                        4,
                        8,
                        16
                    ],
                    "resnet_dt": false,
                    "_comment": " that's all"
                }
            ]
          },
        "fitting_net": {
            "neuron": [
                240,
                240,
                240
            ],
            "resnet_dt": true,
            "seed": 1230123,
            "_comment": " that's all"
        },
        "_comment": " that's all"
    },
    "learning_rate": {
        "type": "exp",
        "start_lr": 0.001,
        "stop_lr": 3.51e-08,
        "decay_steps": 4000,
        "_comment": "that's all"
    },
    "loss": {
        "start_pref_e": 0.02,
        "limit_pref_e": 1,
        "start_pref_f": 1000,
        "limit_pref_f": 1,
        "start_pref_v": 0.02,
        "limit_pref_v": 1,
        "_comment": " that's all"
    },
    "training": {
        "systems": "../../init",
        "set_prefix": "set",
        "stop_batch": 800000,
        "batch_size": "auto",
        "seed": 4560456,
        "_comment": "that's all",
        "disp_file": "lcurve.out",
        "disp_freq": 1000,
        "numb_test": 5,
        "save_freq": 1000,
        "save_ckpt": "model.ckpt",
        "disp_training": true,
        "time_training": true,
        "profiling": false,
        "profiling_file": "timeline.json"
    }
}

<Platform>
runs on EHPC-huhehaote, PBS.
training data path: /home/wanghan/research/xiaoyang-Li/Virial_Devi/tmp/init/
Command: dp train input.json

**Steps to Reproduce**
[failsample.zip](https://github.com/deepmodeling/deepmd-kit/files/6568287/failsample.zip)

**Further Information, Files, and Links**

**Comment by njzjz** [comment]
fixed in #685
